docs/community/developer_call_notes/2020_05_07.rst

```diff
@@ -1,2 +1,101 @@
 Developer call May 7, 2020
---------------------------
\ No newline at end of file
+--------------------------
+
+Scribe: `Roman Lutz <https://github.com/romanlutz>`_
+
+Attendees:
+
+- `Andrew Anderson <https://www.linkedin.com/in/andrewanderson05/>`_
+- `Matthijs Brouns <https://github.com/mbrouns>`_
+- `Miro Dudik <https://github.com/MiroDudik>`_
+- `Richard Edgar <https://github.com/riedgar-ms>`_
+- `Parul Gupta <https://github.com/parul100495>`_
+- `Ken Holstein <https://kenholstein.myportfolio.com/>`_
+- `Adrin Jalali <https://github.com/adrinjalali>`_
+- `Abdul Hannan Kanji <https://github.com/hannanabdul55>`_
+- `Roman Lutz <https://github.com/romanlutz>`_
+- `Vanessa Milan <https://www.microsoft.com/en-us/research/people/vmilan/>`_
+- `Kevin Robinson <https://github.com/kevinrobinson>`_
+- `Mehrnoosh Sameki <https://github.com/mesameki>`_
+- `Hanna Wallach <https://www.microsoft.com/en-us/research/people/wallach/>`_
+
+Organizational updates
+^^^^^^^^^^^^^^^^^^^^^^
+
+#. Microsoft & Fairlearn:
+
+    - goal is to be an open source, community driven project
+    - sociotechnical framing is central to the project, careful and precise
+      wording matter
```

Comment:
A couple small suggestions:
* maybe split this off the first two sub bullets separately under `#. Fairlearn project`
* before the last two sub-bullets add one more explanatory sub-bullet "currently many core members are from Microsoft, but the project is an independent open source project" (a bit repetitive, but doesn't hurt)
* it seems that the sub-bullets jumped an extra level (just me being nitpicky)

Reply:
which ones jumped? 
Adding all the changes you suggested except for the last one which I don't understand yet.

Reply:
should look like this now 
![image](https://user-images.githubusercontent.com/10245648/82507364-de265180-9ab6-11ea-9e85-6e77582a1e9c.png)